tbump/main.py

```diff
@@ -158,7 +159,10 @@ def display_hooks(self) -> None:
             return
         ui.info_2("Would execute these hooks")
         for i, hook in enumerate(hooks):
-            ui.info_count(i, len(hooks), ui.bold, hook.name)
+            desc = [ui.bold, hook.name]
+            if hook.after_push:
+                desc += [ui.reset, ui.brown, "(after push)"]
+            ui.info_count(i, len(hooks), *desc)
```

Comment:
hooks will appear unordered, "after push" hooks might appear before normal hooks, not harmful though

Reply:
There's some architecture missing there I think. Let's take some time to discuss this